mindsdb/api/mysql/mysql_proxy/datahub/datanodes/mindsdb_datanode.py

```diff
@@ -172,20 +173,10 @@ def select(self, table, columns=None, where=None, where_data=None, order_by=None
 
                 for key in ele:
                     row[key] = ele[key]['predicted_value']
-                    # +++ FIXME this fix until issue https://github.com/mindsdb/mindsdb/issues/591 not resolved
-                    typing = None
-                    if key in model['data_analysis']:
-                        typing = model['data_analysis'][key]['typing']['data_subtype']
-
-                    if typing == 'Timestamp' and row[key] is not None:
-                        timestamp = datetime.datetime.utcfromtimestamp(row[key])
-                        row[key] = timestamp.strftime('%Y-%m-%d %H:%M:%S')
-                    elif typing == 'Date':
-                        timestamp = datetime.datetime.utcfromtimestamp(row[key])
-                        row[key] = timestamp.strftime('%Y-%m-%d')
-                    elif typing == 'Int':
+                    # FIXME prefer get int from mindsdb_native in this case
+                    if model['data_analysis'][key]['typing']['data_subtype'] == 'Int':
                         row[key] = int(row[key])
```

Comment:
Is this resolved now ? If so, we should close: https://github.com/mindsdb/mindsdb/issues/591

Reply:
that other issue, probably only for custom models